src/PrestaShopBundle/Utils/ZipManager.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace PrestaShopBundle\Utils;
+
+use RecursiveIteratorIterator;
+use RecursiveDirectoryIterator;
+
+class ZipManager
+{
+    public function createArchive($filename, $folder)
```

Comment:
Hum... returns the filename make no sense as it is already part of the function argument.
Return true or false won't help too.. the main issue here is that the [ZipArchive](http://php.net/manual/fr/ziparchive.open.php) is a pain.

I agree to improve it, but in a dedicated pull request to make this one focused on translation of themes.